calling-apis/managing-asynchronous-calls.md

```diff
@@ -309,11 +334,17 @@ Which of the following most accurately describes where we print the results from
 ### !end-challenge
 <!-- prettier-ignore-end -->
 
-This solution successfully uses the results of one API call to make another API call, but it has a few drawbacks:
+Raffy chose to create a new function `getLocationFromQuery` to manage calling the existing functions `findLatitudeAndLongitude` and `findLocation`. He could have called the second method directly from inside the first with `.then` chaining, but doing so would have linked these two functions together, making it harder to use one without the other.
+
+To use `findLatitudeAndLongitude` from another function and pass on its `response` data to another network call, we needed to make a couple updates in `findLatitudeAndLongitude`: 
+- `return` the result of the `axios` call which returns the promise chain the axios call creates.
```

Comment:
This sentence is a little challenging for me to follow, but maybe because it's Friday afternoon and the sun is shining. 

Trying to think how it could be reworded.

"`return` the result of the `axios` call because we'll need to access the promise chain the axios call creates."


Not sure if that's any more clear. So maybe we can refer to the linked repo for line numbers since code blocks in Learn don't have line numbers instead?

```suggestion
- `return` the result of the `axios` call which returns the promise chain the axios call creates (see line 86 in the linked repo)
```